dqc/system/mol.py

```diff
@@ -294,6 +294,27 @@ def getparamnames(self, methodname: str, prefix: str = "") -> List[str]:
         else:
             raise KeyError("Unknown methodname: %s" % methodname)
 
+    def make_copy(self, **kwargs) -> "Mol":
+        # create dictionary of all parameters
```

Comment:
Could you please add a little bit of docstr to explain about the `kwargs` (e.g. it accepts the same kwargs as `Mol`) or list them. The docstr could be here or in the base class, whichever you prefer.

Reply:
I have tried - please let me know if you would like the language changed at all

Reply:
There are several minor comments, but I'm happy with the language